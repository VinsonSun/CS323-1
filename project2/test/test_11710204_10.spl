int test_11710204_r5()
{
  int a = 10;
  int b = a[10];
  return b;
}
