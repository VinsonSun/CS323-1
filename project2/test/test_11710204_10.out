Error type 10 at Line 4: indexing on non-array variable
